helm/ingress-nginx/templates/_params.tpl

```diff
@@ -60,6 +60,9 @@
 {{- if .Values.controller.enableTopologyAwareRouting }}
 - --enable-topology-aware-routing=true
 {{- end }}
+{{- if .Values.controller.disableLeaderElection }}
+- --disable-leader-election=true
+{{- end }}
```

Comment:
Do not add that as it requires changes which are not contained in the v1.10.0 image.